pkg/drivers/hyperv/powershell.go

```diff
@@ -12,22 +12,16 @@ import (
 	log "github.com/code-ready/crc/pkg/crc/logging"
 )
 
-var powershell string
```

Comment:
It's some form of memoization, `LookPath` is not 100% free. I don't mind not caching it.